harvard-lts/CURIOSity
# semver bump

**semver bump**
* * *

Updating SemVer versions

# How should this be tested?

Pull this branch locally, build, and confirm things work! This is also deployed to dev.
# Test coverage
Yes/No: Are changes in this pull-request covered by:
- unit tests?
- integration tests?

# Interested parties
Tag (@ mention) interested parties
